kafka-zookeeper/Dockerfile

```diff
@@ -0,0 +1,38 @@
+FROM alpine:3.12 AS kafka
```

Comment:
Curious, why is this needed?

Reply:
This is a temp docker container which is used to fetching and extracting kafka tar. Later, we will copy the extracted folder to the final container image. This won't contribute any layer in the final image.

Reply:
Got it. thanks